lintercat/lintercat.com
# Task/remove unnecesary stuff

### What does this PR do?
- Remove contact link
- Remove project type from it's technical card

### Where should the reviewer start?
`/src/components/Project/components/Project-technical-card.vue`
`/src/data/projects.js`

### Does this add new dependencies? (Y/n)
n

### How should this be manually tested?
1. Open the root route
2. There should not be a "contact us" link
3. Open a project
4. There should not be a "INTERNAL PROJECT" or similar above the project name

### Any background context you want to provide?
n
